lib/linter/linter.js

```diff
@@ -1970,6 +2069,14 @@ class Linter {
         return internalSlotsMap.get(this).lastSourceCode;
     }
 
+    getTimes(filename) {
+        return internalSlotsMap.get(this).times[filename] || { passes: [] };
```

Comment:
Maybe another spot where `??` is more appropriate?
```suggestion
        return internalSlotsMap.get(this).times[filename] ?? { passes: [] };
```